games-puzzle/monsterz

```diff
@@ -0,0 +1,45 @@
+SUMMARY="A little arcade puzzle game."
+DESCRIPTION="The goal of the game is to create rows of similar monsters, either horizontally or vertically.\
+The only allowed move is the swap of two adjacent monsters, on the condition that it creates a row of three \
+or more. When alignments are cleared, pieces fall from the top of the screen to fill the board again. Chain \
+reactions earn you even more points."
+HOMEPAGE="http://sam.zoy.org/monsterz/"
+COPYRIGHT="?"
+LICENSE="ISC"
+REVISION="1"
+SOURCE_URI="?"
```

Comment:
```
SOURCE_URI="?"
```
@vrindag, you can't submit this :)
You're supposed to test the recipe. 
Without a correct `SOURCE_URI` you can't even kick a build!

Reply:
Sorry, I though you had clicked Submit on the GCI site.
That's OK, then :)

Reply:
I just claimed the task, then I had tons of homework to finish. Sorry!

Reply:
No problem. Sorry about my inappropriate message.
May I delete this comment and the previous four too? If that's OK for you, you can add a thumbs up and I'll understand.
(This would allow to remove 5 messages in this discussion, and I'd also feel much better without my comment :)

Once again, I'm really sorry about my comment. BTW, you are perfectly right to do your homework before anything else.

Reply:
That's completely fine with me. I just finished my homework, so lots more time for me to build recipes!!!

Reply:
Sorry for the late reply.